src/vm/dllimportcallback.cpp

```diff
@@ -1408,16 +1420,14 @@ VOID UMThunkMarshInfo::RunTimeInit()
         new (&sigInfo) PInvokeStaticSigInfo(pMD);
     else
         new (&sigInfo) PInvokeStaticSigInfo(GetSignature(), GetModule());
-
     if (sigInfo.GetCallConv() == pmCallConvCdecl)
     {
-        // caller pop
-        m_cbRetPop = 0;
+        m_cbRetPop = cbRetPop;
```

Comment:
Is this correct? I would except cdecl to be always zero, no matter whether there is a return buffer argument.